crates/miden-lib/asm/miden/kernel_proc_offsets.masm

```diff
@@ -5,71 +5,78 @@
 # Entire account commitment
 const.ACCOUNT_GET_INITIAL_COMMITMENT_OFFSET=0
 const.ACCOUNT_COMPUTE_CURRENT_COMMITMENT_OFFSET=1
+const.ACCOUNT_COMPUTE_DELTA_COMMITMENT_OFFSET=2
 
 # ID
-const.ACCOUNT_GET_ID_OFFSET=2
+const.ACCOUNT_GET_ID_OFFSET=3
 
 # Nonce
-const.ACCOUNT_GET_NONCE_OFFSET=3                 # accessor
-const.ACCOUNT_INCR_NONCE_OFFSET=4                # mutator
+const.ACCOUNT_GET_NONCE_OFFSET=4                      # accessor
+const.ACCOUNT_INCR_NONCE_OFFSET=5                     # mutator
 
 # Code
-const.ACCOUNT_GET_CODE_COMMITMENT_OFFSET=5
+const.ACCOUNT_GET_CODE_COMMITMENT_OFFSET=6
 
 # Storage
-const.ACCOUNT_GET_STORAGE_COMMITMENT_OFFSET=6
-const.ACCOUNT_GET_ITEM_OFFSET=7
-const.ACCOUNT_SET_ITEM_OFFSET=8
-const.ACCOUNT_GET_MAP_ITEM_OFFSET=9
-const.ACCOUNT_SET_MAP_ITEM_OFFSET=10
+const.ACCOUNT_GET_STORAGE_COMMITMENT_OFFSET=7
+const.ACCOUNT_GET_ITEM_OFFSET=8
+const.ACCOUNT_SET_ITEM_OFFSET=9
+const.ACCOUNT_GET_MAP_ITEM_OFFSET=10
+const.ACCOUNT_SET_MAP_ITEM_OFFSET=11
 
 # Vault
-const.ACCOUNT_GET_VAULT_ROOT_OFFSET=11
-const.ACCOUNT_ADD_ASSET_OFFSET=12
-const.ACCOUNT_REMOVE_ASSET_OFFSET=13
-const.ACCOUNT_GET_BALANCE_OFFSET=14
-const.ACCOUNT_HAS_NON_FUNGIBLE_ASSET_OFFSET=15
+const.ACCOUNT_GET_VAULT_ROOT_OFFSET=12
+const.ACCOUNT_ADD_ASSET_OFFSET=13
+const.ACCOUNT_REMOVE_ASSET_OFFSET=14
+const.ACCOUNT_GET_BALANCE_OFFSET=15
+const.ACCOUNT_HAS_NON_FUNGIBLE_ASSET_OFFSET=16
 
 # Procedure introspection
-const.ACCOUNT_WAS_PROCEDURE_CALLED_OFFSET=16
+const.ACCOUNT_WAS_PROCEDURE_CALLED_OFFSET=17
 
 ### Faucet ######################################
-const.FAUCET_MINT_ASSET_OFFSET=17
-const.FAUCET_BURN_ASSET_OFFSET=18
-const.FAUCET_GET_TOTAL_FUNGIBLE_ASSET_ISSUANCE_OFFSET=19
-const.FAUCET_IS_NON_FUNGIBLE_ASSET_ISSUED_OFFSET=20
+const.FAUCET_MINT_ASSET_OFFSET=18
+const.FAUCET_BURN_ASSET_OFFSET=19
+const.FAUCET_GET_TOTAL_FUNGIBLE_ASSET_ISSUANCE_OFFSET=20
+const.FAUCET_IS_NON_FUNGIBLE_ASSET_ISSUED_OFFSET=21
 
 ### Note ########################################
 # assets
-const.NOTE_GET_ASSETS_INFO_OFFSET=21             # accessor
-const.NOTE_ADD_ASSET_OFFSET=22                   # mutator
+const.NOTE_GET_ASSETS_INFO_OFFSET=22                  # accessor
+const.NOTE_ADD_ASSET_OFFSET=23                        # mutator
+const.INPUT_NOTE_GET_ASSETS_INFO_BY_INDEX_OFFSET=24
+const.OUTPUT_NOTE_GET_ASSETS_INFO_BY_INDEX_OFFSET=25
```

Comment:
nit: I'm not sure we need the `BY_INDEX` part.

Also, I'd probably put them into a new subsection - something like "note introspection" (could go after "note parameters").